File: Volume 10 (1000 - 1099)/1035/1035.py
l = [int(i) for i in input().split(' ')]
if l[1] > l[2] and l[3] > l[0] and l[2] + l[3] > l[0] + l[1] and l[2] > 0 and l[3] > 0 and l[0] % 2 == 0:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")

File: Volume 10 (1000 - 1099)/1035/1035.c
#include <stdio.h>
 
int main() {
    int l[4];
    scanf("%d %d %d %d", &l[0], &l[1], &l[2], &l[3]);
    if (l[1] > l[2] && l[3] > l[0] && l[2] + l[3] > l[0] + l[1] && l[2] > 0 && l[3] > 0 && l[0] % 2 == 0) {
        printf("Valores aceitos\n");
    }
    else {
        printf("Valores nao aceitos\n");
    }
    return 0;
}